### ISSUE TITLE
Migrating large files folder

### ISSUE THREAD

**Issue by kevinjbayer** [directus-community/migration-tool#80]
Working on an upgrade with this tool and I've got about 90GB of files to migrate. If I can just move the original folder to the new one, would it be possible to modify the files.js script to create the new record in the database, but not actually handle copying the file to the new location?

With my first test, it looks like it rewrites the filename to the UUID in the new v9 location, so my initial reaction is that I'll need to have them all copied over.

Any additional insight would be appreciated.

**Comment by rijkvanzanten** [comment]
Great question! I don't think there's a much better way 🤔 The filename on disk needs to follow the format as expected by v9, and the database needs to be updated from integer to UUID primary keys 🤔

**Comment by kevinjbayer** [comment]
Yeah, I don't think my method will work either because of the filename on disk changing. I'm going to try on a larger hard drive and I'm assuming I'll need to use the cookie auth to keep the requests open for longer than the token will probably allow. Time to dig into that for v8.

I'll keep this ticket open for another day or two just in case anyone has any other ideas.
